apps/web/src/app/(admin)/backoffice/rewards/page.tsx

```diff
@@ -0,0 +1,197 @@
+'use client'
```

Comment:
This hurts!! 

Never put a full page as `client` 🙏

Reply:
But it's not a good practice. Why matters is back office? Is the same effort and is more correct

Reply:
it's just a ROI calculation of spending time refactoring backoffice vs spending time doing user-facing product. I agree it's better not to use use client on pages but i prefer we put a todo and move on to more important things rather than spending one hour fixing this atm.